tests/snuba/search/test_backend.py

```diff
@@ -1440,6 +1443,60 @@ def test_sort_trend(self):
         # These will be arbitrarily ordered since their trend values are all 0
         assert set(results[2:]) == set([self.group2, no_before_group, no_after_group])
 
+    def test_sort_inbox(self):
+        start = self.group1.first_seen - timedelta(days=1)
+        inbox_group_1 = self.store_event(
+            data={
+                "fingerprint": ["put-me-in-group1"],
+                "event_id": "2" * 32,
+                "message": "something",
+                "timestamp": iso_format(self.base_datetime),
+            },
+            project_id=self.project.id,
+        ).group
+        inbox_1 = add_group_to_inbox(inbox_group_1, GroupInboxReason.NEW)
+        self.store_event(
+            data={
+                "fingerprint": ["put-me-in-group2"],
+                "event_id": "3" * 32,
+                "message": "something",
+                "timestamp": iso_format(self.base_datetime),
+            },
+            project_id=self.project.id,
+        ).group
+
+        inbox_group_2 = self.store_event(
+            data={
+                "fingerprint": ["put-me-in-group3"],
+                "event_id": "4" * 32,
+                "message": "something",
+                "timestamp": iso_format(self.base_datetime),
+            },
+            project_id=self.project.id,
+        ).group
+        inbox_2 = add_group_to_inbox(inbox_group_2, GroupInboxReason.NEW)
+        inbox_2.update(date_added=inbox_1.date_added - timedelta(hours=1))
+
+        results = self.make_query(
+            [self.project],
+            sort_by="inbox",
+            date_from=start,
+            search_filter_query="is:for_review",
```

Comment:
Should we also have a test for `is:unresolved is:for_review owner:me_or_none `? I assume this is the search that we're going to be running by default and wanting to sort by inbox date

Reply:
I don't know if we need a test for the whole inbox query, it's likely to change over time. I did add in `is:unresolved` just to prove the sort works fine when using other terms